pa11ycrawler/pipelines.py

```diff
@@ -223,7 +232,12 @@ def process_item(self, item, spider):
                 )
             )
 
-        self.check_title_match(item['page_title'], stdout, spider.logger)
+        if stdout:
+            pa11y_results = json.loads(stdout.decode('utf8'))
```

Comment:
I made pa11ycrawler Python 3 compatible [in an earlier pull request](https://github.com/edx/pa11ycrawler/pull/18), partially because Python 3 has stricter standards about bytes vs unicode. In Python 3, the `json` module will only load unicode strings, not bytes, so you _have_ to decode before calling `json.loads()` -- which is exactly the right behavior as per Ned's unicode sandwich. Once we've converted the bytes into unicode, it's *all* unicode, so we can handle special characters in titles (and anywhere else) just fine.